libs/verifiable-credentials/src/test/java/uk/gov/di/ipv/core/library/verifiablecredential/validator/VerifiableCredentialValidatorTest.java

```diff
@@ -56,7 +56,7 @@ static void setup() throws ParseException {
         TEST_SIGNING_KEY = ECKey.parse(EC_PUBLIC_JWK);
         TEST_SIGNING_KEY2 =
                 ECKey.parse(
-                        "{\"crv\":\"P-256\",\"d\":\"o1orSH_mS3u1zzi4wXa9C-cgY2bPyZWN5DxK78JCN6E\",\"kty\":\"EC\",\"x\":\"LziA3lV476BwPG5glvLLx8-FzMbeX2ti9wYlhwCWNhQ\",\"y\":\"NfvgSlu1TMNjjMRM3um29Tv79C4NL8x6WEY7t4BBneA\"}");
+                        "{\"crv\":\"P-256\",\"d\":\"o1orSH_mS3u1zzi4wXa9C-cgY2bPyZWN5DxK78JCN6E\",\"kty\":\"EC\",\"x\":\"LziA3lV476BwPG5glvLLx8-FzMbeX2ti9wYlhwCWNhQ\",\"y\":\"NfvgSlu1TMNjjMRM3um29Tv79C4NL8x6WEY7t4BBneA\"}"); // pragma: allowlist secret
```

Comment:
So is the 'secret' matching against `TEST_SIGNING_KEY2` rather than the key value itself? That's annoying :(

If they're unique, then I think an allowlist comment is probably better anyway.

Reply:
It's matching against the value itself which is typically unique. We can't match it using the name of the variable it's assigned to unless the whole thing is on one line in which case we can use something like the `exclude-line` filter to match against the name of the variable it's assigned to

Reply:
I think we explicitly _don't_ want to match against the variable name - there's nothing guaranteeing that a variable named 'test_key' is not actually a genuine secret!

If it's unique then I think the allowlist comment is the best approach, then someone has to explicitly decide that it's safe each time.